tokenauth/token.go

```diff
@@ -196,9 +233,14 @@ func (t *TokenAuth) ValidateScopeClaim(claims *Claims, requiredScope string) err
 }
 
 // NewTokenAuth creates and configures a new TokenAuth instance
-func NewTokenAuth(acceptRokwireTokens bool, authService *authservice.AuthService) (*TokenAuth, error) {
+func NewTokenAuth(acceptRokwireTokens bool, authService *authservice.AuthService, blacklistSize int) (*TokenAuth, error) {
```

Comment:
I think we should treat `blacklistSize` like the refresh freq params (ie. set default in `NewTokenAuth` and provide a helper function to set it)